index.html

```diff
@@ -1,26 +1,125 @@
+<!-- working on send/add messages, see all messages and delete messages and give error
+ error message if not both name and message is filled in-->
+
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <title>Welcome to CYF Chat</title>
   </head>
+
   <body>
-    <h1>
-      CYF Chat
-    </h1>
-    <h2>
-      Send a message
-    </h2>
+    <h1>CYF Chat</h1>
+    <h2>Send a message</h2>
     <form action="/messages" method="post">
       <p>
         Name: <input type="text" name="from" placeholder="Your Name" /> <br />
         Message: <input type="text" name="text" placeholder="The message..." />
         <br />
       </p>
-      <button type="submit">
-        Send
-      </button>
+      <button type="submit">Send</button>
     </form>
-
+    <br />
     <a href="/messages">See all messages</a>
+
+    <form>
+      <p>
+        Delete Message:
+        <input
+          type="number"
+          id="deleteId"
+          name="id"
+          placeholder="Delete message by ID..."
+        />
+      </p>
+      <button type="button" onclick="deleteMessage()">Submit</button>
+    </form>
+
+    <script>
```

Comment:
Just as a suggestion, we can create a separate javascript file, for example `script.js` and then refer or include that exact javascript file into the script HTML element. You can read more on that [here](https://www.w3schools.com/tags/att_script_src.asp) for example. This way, we can effectively structure our code.